targets/PythonSdk/source/source/PlayFabManualTest.py

```diff
@@ -0,0 +1,57 @@
+# idea here is to include PlayFabSettings, set them,
+# include API.py (or PlayFabSErverAPI.py) and see if you can call a function out of it
+import PlayFabSettings
+import PlayFabEntityAPI
+import PlayFabServerAPI
+import PlayFabClientAPI
+import PlayFabManualTestSettings
+
+PlayFabSettings.TitleId = PlayFabManualTestSettings.TitleId
+PlayFabSettings.SdkVersionString = PlayFabManualTestSettings.SdkVersionString
+customId = PlayFabManualTestSettings.customId
+
+request = {}
+
+request["CreateAccount"] = True
+request["CustomId"] = customId
+request["TitleId"] = PlayFabSettings.TitleId
+request["LoginTitlePlayerAccountEntity"] = True
+
+customData = ""
```

Comment:
customData is very useful for the customer, but it should also default to None, and be optional.

Reply:
This file should not define customData at all, and we should never provide the parameter.